keyboards/planck/rev6/rev6.c

```diff
@@ -14,11 +14,116 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include "rev6.h"
+#include "qwiic.h"
+#include "action_layer.h"
+#include "matrix.h"
+
+#ifdef QWIIC_MICRO_OLED_ENABLE
+
+static uint8_t layer;
+static bool queue_for_send = false;
+static uint8_t encoder_value = 32;
+
+void draw_ui(void) {
```

Comment:
Seems like this is hardware specific. Does this code need to go at the keyboard level or could it be more generic?